roles/openshift_node/tasks/node_system_container.yml

```diff
@@ -16,19 +16,21 @@
   register: update_result
   changed_when: "'Extracting' in update_result.stdout"
   when:
-  - (openshift.common.version is defined) and (openshift.common.version == openshift_version) and ("node" in result.stdout) | bool
+  - l_is_same_version
+  - ("node" in result.stdout) | bool
```

Comment:
Shouldn't need the bool filter here, you should be able to just use `'node' in result.stdout`